root/static/scripts/release-editor/actions.js

```diff
@@ -69,6 +75,83 @@ const actions = {
     }));
   },
 
+  duplicateReleaseGroups: ko.observableArray([]),
+  loadingDuplicateReleaseGroups: ko.observable(false),
+  failedLoadingDuplicateReleaseGroups: ko.observable(false),
+  duplicateReleaseGroupsRequest: null,
+  duplicateReleaseGroupsQuery: null,
+
+  /**
+   * Query for existing release groups that are credited to any of the
+   * currently-credited artists and have similar names to the release.
+   */
+  loadDuplicateReleaseGroups() {
+    const release = this.rootField.release.peek();
+    const query = utils.constructLuceneFieldConjunction({
+      arid: release.artistCredit().names
+        .map((a) => a.artist?.gid)
+        .filter(Boolean),
+      releasegroup: [utils.escapeLuceneValue(release.name() ?? '')],
+    });
+    if (query === this.duplicateReleaseGroupsQuery) {
+      return;
+    }
+
+    // Cancel any in-progress lookup and clear existing results.
+    this.duplicateReleaseGroupsRequest?.abort();
+    this.duplicateReleaseGroupsRequest = null;
+    this.loadingDuplicateReleaseGroups(false);
```

Comment:
I wasn't updating this here before, but I think it makes sense to do so since the request is also being aborted and cleared.